### DISCUSSION TITLE
Easiest way to read Meta data in Nuxt 3?

### DISCUSSION THREAD

**Discussion by cshomo11** [nuxt/nuxt#25422] [Questions]
This might be a dumb question but I've read the docs a lot for Nuxt 3 and there's a lot about how to set site meta data. But is there any easy way to read the existing site meta to pull info from? We tend to reuse some data from the meta. I mean I can do a querySelector search for the meta tags but I just wanted to check if I wasn't missing something simple.

**Comment by ndragun92** [comment]
This is one of ways getting it by page name. But you can also use page path if you like

https://stackblitz.com/edit/github-jmcpgi?file=pages%2Fabout%2Findex.vue

Hope it helps

**Reply by cshomo11** [reply]
Ah I was looking a way to pull from the SEO Meta. So if I set something with `useSeoMeta` I could pull it back out without just reseting the metadata. Right now I'm just doing a querySelector on the page to find the meta property attribute.

I appreciate the response though.
